main.py

```diff
@@ -32,26 +36,55 @@ def setup_tokens():
             )
 
 
+def setup_logging(logging_level=logging.WARNING):
+    # Create logging folder path if doesn't exist
+    logging_directory = "./logs/"
+    if not os.path.exists(logging_directory):
+        os.makedirs(logging_directory)
+
+    logFormatter = logging.Formatter(
+        "%(asctime)s - [%(threadName)s] - [%(levelname)s] - [%(name)s] - %(message)s")
+    rootLogger = logging.getLogger()
+    rootLogger.setLevel(logging_level)
+    # TimedRotating FileHandler
+    timedRotatingFileHandler = TimedRotatingFileHandler(
+        logging_directory + "system_logs.log", when="w0", interval=1)
+    timedRotatingFileHandler.setFormatter(logFormatter)
+    rootLogger.addHandler(timedRotatingFileHandler)
+    # Writes logs to sys.sterr
+    consoleHandler = logging.StreamHandler()
+    consoleHandler.setFormatter(logFormatter)
+    rootLogger.addHandler(consoleHandler)
+
+
 def main():
     setup_tokens()
 
+    # To enable logging on INFO level, logging_level=logging.INFO. Default is ERROR
+    setup_logging(logging_level=logging.INFO)
     clan_members_ranker = ClanMembersRanker()
     royale_api_website_scraper = RoyaleApiWebsiteScraper()
     activity = discord.Game(name="!bothelp")
     bot = Bot("!", activity=activity)
 
     @bot.event
     async def on_ready():
-        print('Bot is up and ready. We have logged in as {0.user}'.format(bot))
+        print(f'Bot is up and ready. We have logged in as {bot.user}')
```

Comment:
/supernit since you're sending/logging the same msg, you could just put it to a variable and then use it lmao
```
on_ready_msg = f'Bot is up and ready. We have logged in as {bot.user}'
print(on_ready_msg)
logging.info(on_ready_msg)
```